python_code_style.md

```diff
@@ -39,39 +85,21 @@ wget https://raw.githubusercontent.com/populationgenomics/team-docs/main/pylint\
 /github-workflow.yaml -O .github/workflows/main.yaml
 ```
 
-This will make GitHub run Pylint on every push and pull request, and display
-checks in the web interface.
+This will make GitHub run the linters on every push and pull request, and 
+display checks in the web interface.
 
 <img src="figures/github_pylint_check.png" width="400"/>
 
-## Running Pylint
-
-To install Pylint into your environment, run:
-
-```bash
-pip install pylint pylint-quotes
-```
-
-`pylint-quotes` is a plugin to Pylint that adds checks of the consistency of
-quotes (we stick to single quotes, with the only reason to prefer them over
-double quotes being the visual noise).
-
-To run Pylint manually, you can use:
-
-```bash
-pylint **/*.py
-```
-
-## Visual Studio Code
+### Visual Studio Code
 
 Visual Studio Code auto-detects a .pylintrc file in the project root and asks
 the user whether to install pylint, if it's not installed already.
 
-## PyCharm
+### PyCharm
 
-1. Make sure you have pylint installed with pip or conda.
+1. Make sure you have pylint installed with `pip install pylint pylint-quotes`.
 
-2. Install a PyCharm "Pylint" plugin ([Preferences] > [Plugins] > search for
+2. Install the PyCharm "Pylint" plugin ([Preferences] > [Plugins] > search for
    "Pylint"). It will automatically find the pylint executable. A new tool
    window called Pylint will be added. You can use it to scan the project:
 
```

Comment:
GitHub doesn't let me comment on line 112, so I'm adding a drive-by nit here ;).

Could you put the open parens for `(Cmd+K)` on the same line to avoid the extra space, please?

Reply:
Sigh, automatic hard wraps aren't smart enough :)